UI/aoc-app/src/App.js

```diff
@@ -1,13 +1,27 @@
 import React from 'react';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Redirect,
+  Route,
+  Switch,
+} from "react-router-dom";
+import CreateProfile from "./Components/CreateProfile";
 
 
 // rendering app
 function App() {
   return (
-    <div className="App">
-      Hello World!   
-    </div>
+    <Router>
+      <div className="App">
+        <Route exact path="/">
+        Hello World!   
```

Comment:
Will the landing page from task 4 be inserted into this route "/"?

Reply:
Yes, we decide what to render with the specified path(s).